templates/travel/partials/results/results.snip.html

```diff
@@ -143,6 +142,7 @@
         </div>
         [[={{ }}=]]
       </template>
+      <span class="force-render">&nbsp;</span>
```

Comment:
You can prob change this to a `div` and drop the `nbsp`, the reviewer at Google hates `nbsp`'s with his life for some reason

Reply:
But if you can't then it's all good, I'll deal with it during the review if it comes up

Reply:
Yeah, I can probably make it a div that's 1px high, will give that a try before I merge this.
It doesn't seem to work if it's just a div with nothing in it.
Super weird stuff.

Reply:
Thanks for making me revisit this, works now as a div 👍